lblod/frontend-lokaal-mandatenbeheer
# Lmb 1010: Global system message

## Description

We want to set a global system message in our apps from an admin panel. Here we can set a message and activate or disable it from showing. This functionality is only available for admins

## How to test

- Login as an admin (search eenheid "bestuur")
- Go to the admin panel>global-system-message 
- Set unset a message and go to any bestuurseenheid and see if the system message is available
- The message shoulde be reactive

- The admin panel card/main menu item is only visible when you are an admin and not impersonating  
- Redirected the admin panel route directly to the system message as we only have one route there and I wasn't sure what to put on the overview page. (possibly the loket cards as in the module overview?)

## Links to other PR's

- https://github.com/lblod/app-lokaal-mandatenbeheer/pull/275
